tools/add_triage_signature.py

```diff
@@ -1057,6 +1057,57 @@ def _update_ticket(self, url, issue_key, should_update=False):
             self._update_triaging_ticket(issue_key, report, should_update=should_update)
 
 
+class WrongBootOrderSignature(ErrorSignature):
+    """
+    """
+
+    ERROR_PATTERN = "a manual booting from installation disk"
+    EVENT_PATTERN = "please boot the host"
+
+
+    def __init__(self, jira_client):
+        super().__init__(jira_client, signature_label="wrong_boot_order",
+                         comment_identifying_string="h1. Wrong boot order")
+
+    def _update_ticket(self, url, issue_key, should_update=False):
+        url = self._logs_url_to_api(url)
+
+        md = get_metadata_json(url)
+
+        cluster = md['cluster']
+        status_info = cluster['status_info']
+
+        if self.ERROR_PATTERN not in status_info:
+            return
+
+        cluster_id = cluster['id']
+        report = ""
+        events = get_events_json(cluster_url=url, cluster_id=cluster_id)
+        reboot_events_by_host = defaultdict(list)
+        for event in events:
+            if self.EVENT_PATTERN in event["message"]:
+                reboot_events_by_host[event["host_id"]].append((event))
+
+        hosts = []
+        for host in cluster['hosts']:
+            events = reboot_events_by_host[host["id"]]
+            if len(events) != 0:
+                hosts.append(OrderedDict(
+                    id=host['id'],
+                    hostname=self._get_hostname(host),
+                    message=events[0]["message"],
+                ))
+
+        if len(hosts) > 0:
+            report = dedent("""
+            Events for rebooting the host from the installation disk were found.
+            It usually indicates a wrong boot order that should be addressed by the user.
+            Can be marked as caused by issue [MGMT-4112|https://issues.redhat.com/browse/MGMT-4112]
```

Comment:
linked issue is "hosts not booting up after..."
maybe we should change the issue / signature naming?

Reply:
I think till now this is the best we have... This one is meant to cover
```
Some of the probable causes:

    Booting order
    Network issues
    Manual intervention by the user

This issue was opened for tracking purposes
```

Other related to wrong boot order are
* https://issues.redhat.com/browse/MGMT-8507
* https://issues.redhat.com/browse/MGMT-3214

Reply:
Maybe we can just remove this line `Can be marked as...`? We keep a list of the tracker-tickets separately and hardcoding it directly in the code is yet another potential for getting out of sync? Not pushing for any direction though, I just want to avoid opening a next bunch of tickets (we already have too many of those)

Reply:
Yeah this ticket is too ambiguous any way, I'll just remove it for now.